api/types/accesslist/member.go

```diff
@@ -90,12 +95,19 @@ func (a *AccessListMember) CheckAndSetDefaults() error {
 		return trace.BadParameter("member name is missing")
 	}
 
-	if a.Spec.Joined.IsZero() || a.Spec.Joined.Unix() == 0 {
-		return trace.BadParameter("member %s joined is missing", a.Spec.Name)
+	var err error
+	if a.Spec.Membership, err = checkInclusion(a.Spec.Membership); err != nil {
+		return trace.Wrap(err, "membership")
 	}
 
-	if a.Spec.AddedBy == "" {
-		return trace.BadParameter("member %s added by is missing", a.Spec.Name)
+	if a.Spec.Membership == InclusionExplicit {
+		if a.Spec.Joined.IsZero() || a.Spec.Joined.Unix() == 0 {
```

Comment:
The `a.Spec.Joined.Unix() == 0` check seems to be not needed: 
```suggestion
		if a.Spec.Joined.IsZero {
```